src/checksum.c

```diff
@@ -1134,7 +1134,11 @@ DWORD WINAPI SumThread(void* param)
 	int num_checksums = CHECKSUM_MAX - (enable_extra_hashes ? 0 : 1);
 
 	if ((image_path == NULL) || (thread_affinity == NULL))
+#if _MSC_VER && !__INTEL_COMPILER
```

Comment:
I'd prefer to have this as a global `EXITTHREAD()` macro defined in `rufus.h` so that we don't have to use that much `#if` boilerplate in the code. Also, my convention for `#if` in Rufus is to go `#if defined(_MSC_VER) && !defined(__INTEL_COMPILER)`, with explicit `define(...)` so please try to follow the same, thank you.